dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```
@@ -315,10 +317,14 @@ export function JsonRpcContextProvider({
           throw new Error(`Missing chain data for chainId: ${chainId}`);
         }
 
-        const valid = _verifyEip155MessageSignature(
-          message,
+        const hashMsg = hashPersonalMessage(message);
+        const rpc =
```

Should we throw an error if there's no rpcProvider for the given chainId ?